.prettierignore

```diff
@@ -1,2 +1,7 @@
-data
-dist
+data/
```

Comment:
Same as for the `.eslintignore`.

Reply:
Not agree here, because for prettier we just need to ignore `data/`, `dist/`, `.*ignore`, `package-lock.json` and `LICENSE` files
`

Reply:
I bumped into a strange behavior of `.prettierignore`.
I have tried this, and it lints only `gs.js` file and does not go deeper.
```
*
!*.js
```
Then I have tried this, and nothing have changed.
```
*
!**/*.js
```

Reply:
The same with `.eslintignore`.

Reply:
@o-rumiantsev, right, I forgot to add the `!*/` rule after the `*`.